pkg/pillar/netmonitor/linux.go

```diff
@@ -151,26 +151,14 @@ func (m *LinuxNetworkMonitor) getInterfaceAttrs(ifIndex int) (attrs IfAttrs, err
 }
 
 func (m *LinuxNetworkMonitor) ifAttrsFromLink(link netlink.Link) IfAttrs {
-	var lowerUP bool
-	switch link.Attrs().OperState {
-	case netlink.OperUnknown:
-		// It is common for cellular modems that the operating state is not reported,
-		// whereas lower-layer IFF_* flags are available and can be used to determine
-		// link status.
-		lowerUP = link.Attrs().RawFlags&unix.IFF_LOWER_UP != 0
-	case netlink.OperUp:
-		lowerUP = true
-	default:
-		lowerUP = false
-	}
 	return IfAttrs{
 		IfIndex:       link.Attrs().Index,
 		IfName:        link.Attrs().Name,
 		IfType:        link.Type(),
 		IsLoopback:    (link.Attrs().Flags & net.FlagLoopback) != 0,
 		WithBroadcast: (link.Attrs().Flags & net.FlagBroadcast) != 0,
 		AdminUp:       (link.Attrs().Flags & net.FlagUp) != 0,
-		LowerUp:       lowerUP,
+		LowerUp:       link.Attrs().RawFlags&unix.IFF_LOWER_UP != 0,
```

Comment:
Please add some comment explaining why we are not using `link.Attrs().OperState` (`IFLA_OPERSTATE`) anymore and have instead resorted to the `IFF_LOWER_UP` flag. We could maybe also reference the kernel docs for this: https://docs.kernel.org/networking/operstates.html

Reply:
(basically just mention what issue we have encountered with `link.Attrs().OperState`)